browser/notifications/BUILD.gn

```diff
@@ -1,22 +1,38 @@
 source_set("notifications") {
   # Remove when https://github.com/brave/brave-browser/issues/10657 is resolved
   check_includes = false
-  if (is_mac) {
-    sources = [
-      "brave_alert_dispatcher_mac.mm",
-      "brave_alert_dispatcher_mac.h",
-      "brave_notification_platform_bridge_mac.mm",
-      "brave_notification_platform_bridge.h",
-    ]
 
-    deps = [
-      "//base",
-      "//chrome/browser/ui/cocoa/notifications:common",
-      "//ui/message_center/public/cpp",
-      "//skia",
-    ]
+  if (is_mac || is_linux || is_win) {
+    if (is_mac) {
+      sources = [
+        "brave_alert_dispatcher_mac.mm",
+        "brave_alert_dispatcher_mac.h",
+        "brave_notification_platform_bridge_mac.mm",
+        "brave_notification_platform_bridge.h",
+        "notification_platform_bridge_brave_custom_notification.h",
+        "notification_platform_bridge_brave_custom_notification.cc"
+      ]
+
+      deps = [
+        "//base",
+        "//chrome/browser/ui/cocoa/notifications:common",
+        "//ui/message_center/public/cpp",
```

Comment:
is there a reason we still need `message_center` after abstracting the code we need?